artemis-core-client/src/main/java/org/apache/activemq/artemis/core/client/impl/ClientSessionImpl.java

```diff
@@ -1539,6 +1540,8 @@ public void end(final Xid xid, final int flags) throws XAException {
             startCall();
             try {
                sessionContext.xaEnd(xid, flags);
+            } catch (ActiveMQNotConnectedException ex) {
+               ActiveMQClientLogger.LOGGER.connectionClosedWarn(ex.getType(), ex.getMessage());
```

Comment:
can you at least keep a logger.debug(ex.getMessage(), ex) here?

This way if a developer ever needs to debug on why.. it would give the developer more information?

Reply:
and as I requested you this small change.. can you rebase against master? that would help me to find that pesky AIO failure on travis.

Reply:
@clebertsuconic Done.